x-pack/plugin/apm/src/main/resources/index-templates/metrics-apm.app.yaml

```diff
@@ -0,0 +1,19 @@
+version: ${xpack.apm.template.version}
+index_patterns: ["metrics-apm.app.*-*"]
+priority: 150
+data_stream: {}
+composed_of:
```

Comment:
@eyalkoren One more list for aligning with the convention

Reply:
FYI, @felixbarny and I have discussed that in the (hopefully not too) long term, this index template should be removed and we should instead write metrics to a non-APM specific data stream, using the generic metrics template. That's why I added histogram & summary dynamic templates to `metrics-mappings.json`.

Reply:
Same with `logs-apm.app`

Reply:
@axw is working on renaming according to the [agreed naming scheme](https://github.com/elastic/elasticsearch/issues/96267)